### ISSUE TITLE
Feedback: Site Title block alignment changes don't work intuitively with layout options

### ISSUE THREAD

**Issue by annezazu** [WordPress/gutenberg#44839]
### Description

Using 6.1 nightly, I noticed that the alignment options (left/right/center) don't stick for the Site Title block. This _feels_ like a bug but some part of me isn't sure if this isn't by design. In general though, it feels odd to be unable to control a very basic part of the Site Title block, a common block folks with FSE will interact with.

### Step-by-step reproduction instructions

1. Use a test site with 6.1 nightly or at least beta 3 and TT2. 
2. Open the Appearance > Editor.
3. Select the Site Title. Try to change the alignment in the block toolbar. Notice nothing happens.

### Screenshots, screen recording, code snippet

https://user-images.githubusercontent.com/26996883/194961539-f4847839-a4be-4123-933f-7aac232efee9.mov

### Environment info

- WordPress 6.1-beta3-54428
- No GB
- TT2 and TT3

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

**Comment by glendaviesnz** [comment]
This is because the site title is in a flex container so the width is constrained to the width of the content, so there is no additional space available to reflect changes in text alignment. If the flex is removed from the wrapper it works as expected:

https://user-images.githubusercontent.com/3629020/197879871-d8912c89-8472-40bb-96e1-eb2eac15a1d5.mp4

but then the horizontal layout of the header is lost. I am not sure if there is any fix for this other than trying to detect if the block has a constrained width and not showing the alignment options if so. The same issue exists in 6.0.3, so I think we should take this off the 6.1 board - what do you think @annezazu ?

**Comment by annezazu** [comment]
Ah ha. Great sleuthing. Agreed on removing for 6.1! Thanks for following up.

**Comment by carolinan** [comment]
I assume we want this check in the text alignment control not only on the site title?

**Comment by ndiego** [comment]
I don't think we should add a check for this because we now have the Width options for blocks in flex containers. There are situations where you would want a fixed width block in a Row and then style the text alignment. The paragraph block is a good example. While the presentation of the text alignment is a bit confusing in the situation presented above, it's needed functionality in my opinion.

**Comment by annezazu** [comment]
I'll convert this to be more about feedback/considerations as this remains a pretty confusing UX.
